open-apparel-registry/open-apparel-registry
# Add alternate country names

## Overview

These missing country names were encountered while processing lists in production.

Connects #1063 

## Testing Instructions

These instructions assume you have run `./scripts/resetdb`

* Download and unzip [country-alias-test.csv.zip](https://github.com/open-apparel-registry/open-apparel-registry/files/4998102/country-alias-test.csv.zip)
* Checkout `develop`
* Log in as c2@example.com
* Browse http://localhost:6543/contribute and upload the test list
* Run the parse process
  * ./scripts/manage batch_process --list-id 16 --action parse
* Browse http://localhost:6543/lists/16?status=ERROR_PARSING&page=1&rowsPerPage=20 and verify that there are 14 failed rows.
* Checkout this branch
* Browse http://localhost:6543/contribute and upload the test list again
* Run the parse process
  * ./scripts/manage batch_process --list-id 17 --action parse
*  Browse http://localhost:6543/lists/17?status=ERROR_PARSING&page=1&rowsPerPage=20 and verify that there are no failed rows.

## Checklist
